_layouts/home_base.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
This seems to be working as expected but do we really want the extra file? It seems we already have `home.html` as a layout, so we could just have the one template contain all layout code.

Reply:
It was to avoid the container margin from the default layout, maybe I should move the part out of the layout.